raiden/network/proxies/token_network.py

```diff
@@ -191,8 +200,17 @@ def _new_netting_channel(self, partner: typing.Address, settle_timeout: int):
         if self.channel_exists_and_not_settled(self.node_address, partner):
             raise DuplicatedChannelError('Channel with given partner address already exists')
 
+        gas_limit = self.proxy.estimate_gas(
+            'openChannel',
+            self.node_address,
+            partner,
+            settle_timeout,
+        )
+        gas_limit = max(gas_limit or 0, GAS_REQUIRED_FOR_OPEN_CHANNEL)
```

Comment:
why is this `gas_limit or 0` needed? I'd rather fix `estimate_gas` to always return an estimate

Reply:
FIne for me